### DISCUSSION TITLE
Are some json.gz files encrypted?

### DISCUSSION THREAD

**Discussion by medlyze** [CMSgov/price-transparency-guide#535] [Q&A] [answered]
We’ve had difficulties opening some json.gz files after unzipping them. They appear to be encrypted. Is there a best practice we should follow to successfully read files after unzipping them? 

We do not have this issue with table of content files that are unzipped .json files. 

Thank you!

**Comment by ghost** [comment, marked as answer]
The files should only be compressed and not encrypted. You should provide a specific example of a file you have downloaded that has been encrypted to allow others to verify and assist you.

**Reply by medlyze** [reply]
Thank you for your reply. As an example:

We’re successfully opening this .json: https://tic-mrf.regence.com/mrf/current/2022-07-01_Regence%20BlueShield-ASO_index.json

However, opening the embedded json.gz is problematic. Example link: location":"https://regencebswa.mrf.bcbs.com/2022-07_020_02I0_in-network-rates.json.gz?&Expires=1663282700&Signature=IOXejooaId2pVJowprohtA8NTYYaOZoKvdx2k0A2wc7PPjdSC9tq7uSQl~7vYV~4FGXsRKe1J1B2zkiK-7h-7OxGsFpTnSpGtAE2SSHIuzFuET6tE8Z~XFTAfQRYE5GSERqCPYDTCbP75-ztXBvz~Tx3c0f~sLVx4mlo-a2ZspdgiwWXja~pv9bOqATA7W6R3xhXNIstC6hP3I9Pu2owSxIyRV-OqlWLAE9isdM2MPI-w3m~EA2SzniQx6SIzAwWgcE52HfC26YLOjHC10aMsOJ76h2Q3qm1ATWtJkgxNUX6Qq8By-P2O-mqN4AZj109HDD5nB2HEBLYG7OYDoGnYg__&Key-Pair-Id=K27TQMT39R1C8A

Very much appreciate your direction on this issue.

**Reply by ghost** [reply]
I was able to download the file based on the link from above and use 7-zip to extract the file into a 6,158,755 KB file. I was able to see the file's data with no issue as it was not encrypted. For example the first set of lines

_D:\work>type 2022-07_020_02I0_in-network-rates.json | more
{"reporting_entity_name":"Arkansas Blue Cross and Blue Shield","reporting_entity_type":"Health Insurance Issuer","last_updated_on":"2022-06-10","version":"1.0.0","provider_references":[{"provider_group_id":20.000000001,"provider_groups":[{"npi":[1003002510],"tin":{"type":"npi","value":"1003002510"}}]},{"provider_group_id":20.000000002,"provider_groups":[{"npi":[1003016700],"tin":{"type":"npi","value":"1003016700"}}]},{"provider_group_id":20.000000003,"provider_groups":[{"npi":[1003017302],"tin":{"type":"npi","value":"1003017302"}}]},{"provider_group_id":20.000000004,"provider_groups":[{"npi":[1003018748],"tin":{"type":"npi","value":"1003018748"}}]},{"provider_group_id":20.000000005,"provider_groups":[{"npi":[1003025594],"tin":{"type":"npi","value":"1003025594"}}]},{"provider_group_id":20.000000006,"provider_groups":[{"npi":[1003025826],"tin":{"type":"npi","value":"1003025826"}}]},{"provider_group_id":20.000000007,"provider_groups":[{"npi":[1003027061],"tin":{"type":"npi","value":"1003027061"}}]},{"provider_group_id":20.000000008,"provider_groups":[{"npi":[1003035338],"tin":{"type":"npi","value":"1003035338"}}]},{"provider_group_id":20.000000009,"provider_groups":[{"npi":[1003036302],"tin":{"type":"ein","value":"87-1737453"}}]},{"provider_group_id":20.000000011,"provider_groups":[{"npi":[1003042292],"tin":{"type":"npi","value":"1003042292"}}]},{"provider_group_id":20.000000012,"provider_groups":[{"npi":[1003044207],"tin":{"type":"npi","value":"1003044207"}}]},{"provider_group_id":20.000000013,"provider_groups":[{"npi":[1003045097],"tin":{"type":"npi","value":"1003045097"}}]},{"provider_group_id":20.000000014,"provider_groups":[{"npi":[1003048943],"tin":{"type":"npi","value":"1003048943"}}]},{"provider_group_id":20.000000015,"provider_groups":[{"npi":[1003052234],"tin":{"type":"npi","value":"1003052234"}}]},{"provider_group_id":20.000000016,"provider_groups":[{"npi":[1003056938],"tin":{"type":"npi","value":"1003056938"}}]},{"provider_group_id":20.000000017,"provider_groups":[{"npi":[1003058850],"tin":{"type":"ein","value":"20-5209319"}}]},{"provider_group_id":20.000000018,"provider_groups":[{"npi":[1003061565],"tin":{"type":"npi","value":"1003061565"}}]},{"provider_group_id":20.000000019,"provider_groups":[{"npi":[1003064460],"tin":{"type":"npi","value":"1003064460"}}]},{"provider_group_id":20.000000021,"provider_groups":[{"npi":[1003069147],"tin":{"type":"npi","value":"1003069147"}}]},{"provider_group_id":20.000000022,"provider_groups":[{"npi":[1003070129],"tin":{"type":"npi","value":"1003070129"}}]},{"provider_group_id":20.000000023,"provider_groups":[{"npi":[1003073628],"tin":{"type":"npi","value":"1003073628"}}]},{"provider_group_id":20.000000024,"provider_groups":[{"npi":[1003080763],"tin":{"type":"npi","value":"1003080763"}}]},{"provider_group_id":20.000000025,"provider_groups":[{"npi":[1003080854],"tin":{"type":"npi","value":"1003080854"}}]},{"provider_group_id":20.000000026,"provider_groups":[{"npi":[1003094244],"tin":{"type":"npi","value":"1003094244"}}]},{"provider_group_id":20.000000027,"provider_groups":[{"npi":[1003100603],"tin":{"type":"npi","value":"1003100603"}}]},{"provider_group_id":20.000000028,"provider_groups":[{"npi":[1003104498],"tin":{"type":"npi","value":"1003104498"}}]},{"provider_group_id":20.000000029,"provider_groups":[{"npi":[1003107160],"tin":{"type":"npi","value":"1003107160"}}]},{"provider_group_id":20.000000031,"provider_groups":[{"npi":[1003110420],"tin":{"type":"npi","value":"1003110-- More  --_

**Reply by medlyze** [reply]
Thank you! We'll revisit using 7-zip and confirm we are able to produce the same results.
